package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "validator",
   "description": "String validation and sanitization",
-  "version": "10.13.0",
+  "version": "10.12.0",
```

Comment:
You can un-stage this change, version change is done by @chriso during publishing, since many other changes will be in by then.